### ISSUE TITLE
Undefined symbols for architecture arm64:   "__swift_FORCE_LOAD_$_swiftCompatibility56"

### ISSUE THREAD

**Issue by antoinedelp** [lottie-react-native/lottie-react-native#1050]
### Description

When running lottie-react-native @5.1.6 with react-native 0.72.2, we can't build the app on iOs. We get the following error message: 

`ld: warning: Could not find or use auto-linked library 'swiftCompatibility56'
ld: warning: Could not find or use auto-linked library 'swiftCompatibilityConcurrency'
Undefined symbols for architecture arm64:
  "__swift_FORCE_LOAD_$_swiftCompatibility56", referenced from:
      __swift_FORCE_LOAD_$_swiftCompatibility56_$_Lottie in Lottie(AnimatedButton.o)
      __swift_FORCE_LOAD_$_swiftCompatibility56_$_Lottie in Lottie(PreCompLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibility56_$_Lottie in Lottie(SolidLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibility56_$_Lottie in Lottie(GradientRenderLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibility56_$_Lottie in Lottie(ShapeContainerLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibility56_$_Lottie in Lottie(TextLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibility56_$_Lottie in Lottie(LottieAnimationViewBase.o)
      ...
     (maybe you meant: __swift_FORCE_LOAD_$_swiftCompatibility56_$_Lottie, __swift_FORCE_LOAD_$_swiftCompatibility56_$_lottie_react_native )
  "__swift_FORCE_LOAD_$_swiftCompatibilityConcurrency", referenced from:
      __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_Lottie in Lottie(AnimatedButton.o)
      __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_Lottie in Lottie(PreCompLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_Lottie in Lottie(SolidLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_Lottie in Lottie(GradientRenderLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_Lottie in Lottie(ShapeContainerLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_Lottie in Lottie(TextLayer.o)
      __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_Lottie in Lottie(LottieAnimationViewBase.o)
      ...
     (maybe you meant: __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_Lottie, __swift_FORCE_LOAD_$_swiftCompatibilityConcurrency_$_lottie_react_native )
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
`

This error also occurs on the "lottie-react-native": "next" version.

**Expected behavior:** : the app should build on iOs

**Actual behavior:** : it does not build.

**Comment by CoreyCole** [comment]
I'm on rn@0.72.3 and lottie-react-native@5.1.6 build error:
```
SwiftCompile normal arm64 .../node_modules/lottie-react-native/src/ios/LottieReactNative/PlatformColor.swift (in target 'lottie-react-native' from project 'Pods')

<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "lottie-react-native-umbrella.h"
        ^
.../ios/Pods/Headers/Public/lottie_react_native/lottie-react-native-umbrella.h:13:9: note: in file included from .../ios/Pods/Headers/Public/lottie_react_native/lottie-react-native-umbrella.h:13:
#import "Bridging.h"
        ^
.../node_modules/react-native/ReactCommon/react/bridging/Bridging.h:10:10: error: 'react/bridging/AString.h' file not found
#include <react/bridging/AString.h>
         ^
<unknown>:0: error: could not build Objective-C module 'lottie_react_native'

** BUILD FAILED **


The following build commands failed:
        SwiftCompile normal arm64 Compiling\ PlatformColor.swift /Users/coreycole/cdev/BlueCanoeReactNative/node_modules/lottie-react-native/src/ios/LottieReactNative/PlatformColor.swift (in target 'lottie-react-native' from project 'Pods')
        SwiftCompile normal arm64 /Users/coreycole/cdev/BlueCanoeReactNative/node_modules/lottie-react-native/src/ios/LottieReactNative/PlatformColor.swift (in target 'lottie-react-native' from project 'Pods')
(2 failures)
```

**Comment by matinzd** [comment]
Are you using XCode beta?

**Comment by matinzd** [comment]
What is your XCode version?

**Comment by antoinedelp** [comment]
Hi @matinzd, my Xcode version is `14.3.1`
And my swift version is 
`swift-driver version: 1.75.2 Apple Swift version 5.8.1 (swiftlang-5.8.0.124.5 clang-1403.0.22.11.100)
Target: arm64-apple-macosx13.0`

**Comment by matinzd** [comment]
Have you tried creating bridging headers for Swift? It seems that swift is not enabled on your project.

Refer to this thread in stackoverflow:

https://stackoverflow.com/a/41788055

**Comment by antoinedelp** [comment]
Yes, I created a bridging-header in this project several years ago already! I'll try to create a new one but I don't think that should be necessary, since lottie is the only package where that Swift error occurs. Other packages requiriung bridging headers compile fine.

**Comment by antoinedelp** [comment]
I recreated a bridge-header file, and made sure to select all the targets of my group. It worked!
Thansk @matinzd

**Comment by KelvinPortuphy** [comment]
@antoinedelp  can you share the steps you performed, including the content of the header file?
